speed_games/results_tool/scripts/index.js

```diff
@@ -0,0 +1,293 @@
+        const candidateScore = candidate.value;
+        const value = (candidateScore - minScore) / (maxScore - minScore)
+
+        const ship = document.createElement('div');
+        const p = document.createElement('p')
+        const img = globalVariables.shipGenerator.generateShipImage();
+
+        ship.style.animationDuration = `${HELPER_FUNCTIONS.randomBetween(1, 3)}s`
+        ship.style.animationDelay = `${HELPER_FUNCTIONS.randomBetween(0, .2)}s`
+        ship.classList.add('ship')
+        ship.style.background = candidate.color
+        img.height = CONFIG.SETTINGS.sprites.h;
+        img.width = CONFIG.SETTINGS.sprites.w;
+        p.innerHTML = candidate.index.toString()
+        ship.append(p)
+        ship.append(img)
+        document.querySelector('#ship-container').append(ship);
+
+        const POSITION = `${((value) * 30) + 2}vh`
+
+        ship.style.top = `calc(90vh - ${CONFIG.SETTINGS.sprites.h}px)`
+        ship.style.left = `${(index * (90 / candidateCount)) + 5}vw`
+        const key = `ship-${index}`
+        ship.id = key
+
+        setTimeout(() => {
+            const r = () => HELPER_FUNCTIONS.randomBetween(0.2, 0.99)
+            const y = () => HELPER_FUNCTIONS.randomBetween(0.2, 0.99)
+            ship.style.transition = `all ${duration}s cubic-bezier(${r()},${r()},${y()},${y()})`
+            // ship.style.transition = `all ${duration}s ease`
+            ship.style.top = POSITION;
+        }, 200)
+    })
+
+    const onEndRace = () => {
+        document.querySelector('.leaderboard').style.display = 'block'
+        const container = document.querySelector('#leaderboard-container')
+        const template = document.querySelector('#leaderboard-template')
+        const sorted = sortBy(x => x.value, Object.assign({}, candidates))
+
+        setTimeout(() => {
+            Array.from(document.querySelectorAll('.ship')).forEach(element => {
+                element.style.transition = 'none'
+                element.style.animation = 'none'
+            })
+        }, 10);
+
+        shootConfetti();
+
+        sorted.forEach((candidate, index) => {
+            const item = template.content.cloneNode(true);
+            item.querySelector('.place').textContent = `${index + 1}. `
+            item.querySelector('.index').textContent = candidate.index.toString()
+            item.querySelector('.index').style.background = candidate.color
+            item.querySelector('.name').textContent = `${candidate.name}: `
+            item.querySelector('.value').textContent = candidate.value.toFixed(2)
+            container.appendChild(item);
+        });
+        settings.callback?.();
+    }
+
+}
+
+document.querySelector("#input").addEventListener("input", (event) => {
+    parseCSV(event.target.files[0], {
+        header: true,
+        complete: inputCallback
+    });
+})
+
+const inputCallback = (raw) => {
+    const data = raw.data
+    data.shift(); // remove header from data
```

Comment:
This will remove data (header row is already removed with `header:true` on line 185).
I'm guessing it's a leftover from an initial version where header: true wasn't present?